darwin/importer/importer.py

```diff
@@ -1400,8 +1400,9 @@ def _overwrite_warning(
     """
     files_to_overwrite = []
     for local_file in local_files:
+        item_id = remote_files.get(local_file.full_path)[0]
```

Comment:
`item_id` is not a requirement to import annotation files. Therefore, read it from the corresponding remote file, where we are guaranteed to find it